pype/settings/entities/schemas/projects_schema/schema_project_nuke.json

```diff
@@ -6,6 +6,32 @@
     "is_file": true,
     "children": [
         {
+            "type": "dict",
+            "collapsible": true,
+            "key": "triggers",
+            "label": "Operation triggers",
+            "children": [
+                {
+                    "type": "dict",
+                    "collapsible": false,
+                    "key": "workfile",
+                    "label": "Workfile related",
+                    "is_group": true,
+                    "children": [
+                        {
+                            "type": "boolean",
+                            "key": "open_at_start",
+                            "label": "Open Workfile window at start of a Nuke session"
+                        },
+                        {
+                            "type": "boolean",
+                            "key": "open_last_version",
+                            "label": "Open last available workfile version or create a vanila script"
```

Comment:
this is duplicating another setting that is working on a global level right now. we'll need to make it accessible to all host eventually, but even if we only do it here for now, the checkbox should only be for `create initial nuke script`, not whether it should be opened or not. 
![image](https://user-images.githubusercontent.com/3333008/111982674-8955cc00-8b09-11eb-82a5-2928e030923a.png)